packages/api/src/brevo-config/brevo-config.resolver.ts

```diff
@@ -0,0 +1,103 @@
+import { AffectedEntity, RequiredPermission, validateNotModified } from "@comet/cms-api";
+import { EntityManager, EntityRepository, wrap } from "@mikro-orm/core";
+import { InjectRepository } from "@mikro-orm/nestjs";
+import { Type } from "@nestjs/common";
+import { Args, ID, Mutation, Query, Resolver } from "@nestjs/graphql";
+
+import { BrevoApiSenderService } from "../brevo-api/brevo-api-sender.service";
+import { BrevoApiSender } from "../brevo-api/dto/brevo-api-sender";
+import { EmailCampaignScopeInterface } from "../types";
+import { DynamicDtoValidationPipe } from "../validation/dynamic-dto-validation.pipe";
+import { BrevoConfigInput, BrevoConfigUpdateInput } from "./dto/brevo-config.input";
+import { BrevoConfigInterface } from "./entities/brevo-config-entity.factory";
+
+export function createBrevoConfigResolver({
+    Scope,
+    BrevoConfig,
+}: {
+    Scope: Type<EmailCampaignScopeInterface>;
+    BrevoConfig: Type<BrevoConfigInterface>;
+}): Type<unknown> {
+    @Resolver(() => BrevoConfig)
+    @RequiredPermission(["brevo-newsletter-config"])
+    class BrevoConfigResolver {
+        constructor(
+            private readonly entityManager: EntityManager,
+            private readonly brevoSenderApiService: BrevoApiSenderService,
+            @InjectRepository(BrevoConfig) private readonly repository: EntityRepository<BrevoConfigInterface>,
+        ) {}
+
+        private async validateSender({ email, name }: { email: string; name: string }): Promise<boolean> {
+            const senders = await this.brevoSenderApiService.getSenders();
+
+            if (!senders || !senders.some((sender) => sender.email === email && sender.name === name)) {
+                return false;
+            }
+
+            return true;
+        }
```

Comment:
In which case do you think this would be a problem when sending? 
If the sender gets changed in brevo directly after setting the sender in the config? 🤔

Reply:
When creating a brevo account there is a default sender set.

I'm not entirely sure but when you add a sender afterwards, this default sender is automatically deleted/deactivated and when sending a test email or a campaign, it just throws strange errors. 

But I'm not sure if this is still the case, I'll check this in the next project